### PR TITLE
#7 Task. Gaziz_Tuleukulov

### PR DESCRIPTION
**Pull request by Gaziz666**
https://travis-ci.org/github/Gaziz666/js-assignments/builds/707337931

### CODE REVIEW

**Comment by elizavetaShcherbatykhAis** [comment]
Your PR should contain files related only to this task, here I see the files of previous tasks. They shouldn't be here. Please remove these files from the PR.
I'll mark the PR as draft please click "ready for review" when it will be finished!
Thank you!

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -82,8 +106,50 @@ function* getFibonacciSequence() {
...
+    /*while (queue.length > i) {
+        let currentNode = queue[i];
+        if(currentNode.children !== undefined) { 
+            for(let j = currentNode.children.length - 1; j >= 0; j--){
+                queue.splice( i+1, 0, currentNode.children[j] );
+            }
+        }
+        yield queue[i]
+        i++;
+    }*/
```
Comment (elizavetaShcherbatykhAis) [inline task/07-yield-tasks.js]: Please don't leave any commented out lines

File: task/07-yield-tasks.js [hunk 2]
```diff
@@ -82,8 +106,50 @@ function* getFibonacciSequence() {
...
+                count.pop()
+                i--;
+                count[i]++
+            }
+        }else{
+            queue.pop();
+            count.pop()
+            i--;
+            count[i]++
+
```
Comment (elizavetaShcherbatykhAis) [inline task/07-yield-tasks.js]: It can be much shorter :)

**Comment by elizavetaShcherbatykhAis** [comment]
Much better :) Please check the other comments too
I'll mark the PR as draft please click "ready for review" when it will be finished!
Thank you!

**Comment by Gaziz666** [comment]
I hope I managed to make it shorter

**Comment by elizavetaShcherbatykhAis** [comment]
Well done! I close PR as accepted

**Pull request closed without merging**